### ISSUE TITLE
Starling 2FA not possible with domain user through PowerShell SDK

### ISSUE THREAD

**Issue by aamburn** [OneIdentity/safeguard-ps#118]
It is not possible to do 2FA with a domain user through PowerShell SDK. 2FA works well with a local user via the SDK and logging in to the UI directly with the domain user using 2FA works well, too, it's just the combination of PowerShell and the domain user that's problematic.

Note that AD 2FA user can log into UI or Swagger just fine; this issue is specific to PS. 

Verbose output follows:
```@{Status=Needs2FA; UserToken=; PrimaryProviderId=ad4; SecondaryProviderId=Starling 2FA}
At C:\Program Files (x86)\WindowsPowerShell\Modules\safeguard-ps\2.5.236\safeguard-ps.psm1:721 char:13
+ throw $local:LoginResponse
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo : OperationStopped: (@{Status=Needs2...d=Starling 2FA}:PSObject) [], RuntimeException
+ FullyQualifiedErrorId : @{Status=Needs2FA; UserToken=; PrimaryProviderId=ad4; SecondaryProviderId=Starling 2FA}
```

**Comment by petrsnd** [comment]
@aamburn I believe this was fixed with PR#120 (https://github.com/OneIdentity/safeguard-ps/pull/120)

I am no longer able to reproduce.  Please confirm and close the issue.

**Comment by aamburn** [comment]
I updated safeguard-ps and was able to verify that this issue is fixed. 

Closing.
